base/applications/rapps/appview.cpp

```diff
@@ -1793,14 +1793,18 @@ VOID CApplicationView::OnCommand(WPARAM wParam, LPARAM lParam)
             switch (NotifyCode)
             {
             case CBN_SELCHANGE:
+                static int nLastCusMode = 0;
```

Comment:
Why is this a static int local to this switch case?

Reply:
No need to have it elsewhere => declaration at the minimal code subset. Is it a wrong practice ?

Reply:
I think it was expected to use a more object-oriented way to proceed.

Reply:
OK I understand. Is it a "must" for this PR (moving static int nLastCusMode to a member of the class) or a recomm for future PR ?